apps/condo/domains/billing/utils/serverSchema/index.js

```diff
@@ -69,15 +70,39 @@ async function registerBillingReceipts (context, data) {
  * @param period {string}
  * @return {Promise<*>}
  */
-const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
-    const payments = await  find('Payment', {
-        organization: { id: organizationId },
-        accountNumber: accountNumber,
-        period: period,
-        status_in: [PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS],
-        recipientBic: bic,
-        recipientBankAccount: bankAccount,
-    })
+const getPaymentsSum = async (context, receiptId, organizationId, accountNumber, period, bic, bankAccount) => {
+    const directReceiptIdWhere = { AND: [
+        { receipt: { id: receiptId } },
+        { receipt_is_null: false },
```

Comment:
You do not need to check receipt_is_null here as it is already receipt: { id: receiptId }